src/openforms/forms/tests/factories.py

```diff
@@ -32,6 +41,37 @@ class Params:
             generate_minimal_setup=False,  # there are no form steps
             is_appointment=True,
         )
+        # prevent options passed to Form() and set a default
+        registration_backend_options = {}
+
+    @factory.lazy_attribute
+    def registration_backend__options(backend_resolver):
+        # Then JIT read it from the parent Params
+        # I've seen self modifying assembly, cpp/c++ template hacks
+        # but this API... 🤯
+        return backend_resolver.factory_parent.registration_backend_options
+
+    @factory.post_generation
+    def registration_backend(form, create, extracted, **kwargs):
+        if extracted is None:
+            return
+        (
+            FormRegistrationBackendFactory.create
+            if create
+            # Does build even make sense? Is it just to keep the garbage collector busy?
+            # Let's anyway; maybe the constructor has side-effects; everything is possible.
+            else FormRegistrationBackendFactory.build
+        )(form=form, backend=extracted, **kwargs)
```

Comment:
just FYI - I think doing `SomeFactory()` will invoke the same build strategy as the parent/root, but i'm not 100% sure. Either way I prefer the explicit approach anyway.